server/src/com/cloud/server/StatsCollector.java

```diff
@@ -251,8 +262,9 @@ public boolean start() {
     }
 
     private void init(Map<String, String> configs) {
-        _executor = Executors.newScheduledThreadPool(4, new NamedThreadFactory("StatsCollector"));
+        _executor = Executors.newScheduledThreadPool(6, new NamedThreadFactory("StatsCollector"));
```

Comment:
I understand that ``6`` is in the ``init`` method.  My question is how did we arrive at that number?  Why is 6 instead of 2 or 10?